src/main/scala/mesosphere/marathon/api/v2/DeploymentsResource.scala

```diff
@@ -63,25 +63,23 @@ class DeploymentsResource @Inject() (
     }
   }
 
-  private def toInfo(
-    deployment: DeploymentPlan,
-    currentStepInfo: DeploymentStepInfo): JsObject = {
+  private def toInfo(deployment: DeploymentPlan, currentStepInfo: DeploymentStepInfo): JsObject = {
+    def deploymentAction(action: DeploymentAction): JsObject =
+      Json.obj (
+        "action" -> action.getClass.getSimpleName,
+        "app" -> action.app.id
+      )
+    def currentAction(action: DeploymentAction): JsObject = deploymentAction(action) ++
+      Json.obj ("readinessChecks" -> currentStepInfo.readinessChecksByApp(action.app.id))
 
-    val steps = deployment.steps.map(step => step.actions.map(actionToMap)).map(Json.toJson(_))
     Json.obj(
       "id" -> deployment.id,
       "version" -> deployment.version,
       "affectedApps" -> deployment.affectedApplicationIds.map(_.toString),
-      "steps" -> steps,
-      "currentActions" -> currentStepInfo.step.actions.map(actionToMap),
+      "steps" -> deployment.steps.map(_.actions.map(deploymentAction)),
+      "currentActions" -> currentStepInfo.step.actions.map(currentAction),
```

Comment:
This doesn't match the format described in #3562 (`currentActions` vs `currentStepInfo`), and probably other changes as well.

Can you please update https://mesosphere.github.io/marathon/docs/rest-api.html#get-v2-deployments showing the full output? It would also be nice to have a test that checks the json output using a hard-coded json example.